src/Gui/Window.h

```diff
@@ -46,6 +46,19 @@ class GuiExport WindowParameter : public ParameterGrp::ObserverType
   /// return the parameter group of this window
   ParameterGrp::handle getWindowParameter();
 
+  /// connect slot to ParameterManager signal
+  template<typename S, typename T>
+  void setSlotParamChanged(S slot, T* obsPtr)
+  {
+    namespace bp = boost::placeholders;
+    if (_handle->Manager()) {
+        connParamChanged = _handle->Manager()->signalParamChanged.connect(
+            boost::bind(slot, obsPtr, bp::_1 ,bp::_2 ,bp::_3 ,bp::_4));
+    }
+  }

```

Comment:
Any reason to continue using `boost::bind`, now that we have `std::bind`? They are basically identical, I believe.

Reply:
No special reason. There is currently a mix between std::bind and boost::bind throughout FreeCAD code.
If you prefer to migrate from boost::bind to std::bind, I'll use the latter here.

Reply:
I think at some point we should migrate, but it's not necessary to change this one unless you want to.